resources/cli-reference.md

```diff
@@ -245,6 +245,10 @@ Delete a local dataset:
 
     kamu delete my.dataset
 
+Delete local datasets matching pattern:
+
+    kamu delete my.dataset.%
+
```

Comment:
Careful here, would conflict with master where I've fixed that already

Reply:
This commit has been dropped